hooks/tk-multi-publish2/nuke/publish_files.py

```diff
@@ -96,7 +96,7 @@ def _sync_frame_range(self, item):
             # compare if the frame range set at root level is same as the shotgun cut_in, cut_out
             root = nuke.Root()
             if root.firstFrame() != data[in_field] or root.lastFrame() != data[out_field]:
-                self.logger.error("Frame range not synced with Shotgun.")
+                self.logger.warning("Frame range not synced with Shotgun.")
                 return False
```

Comment:
You probably want to return True here to unblock Publish. But that would probably show validations passed, in which case the user won't bother to read the warning. Perhaps a pop-up message?